_inc/client/writing/speed-up-site.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { FormFieldset } from 'components/forms';
+import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
+import { getModule } from 'state/modules';
+import { isModuleFound as _isModuleFound } from 'state/search';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { ModuleToggle } from 'components/module-toggle';
+
+const SpeedUpSite = moduleSettingsForm(
+	React.createClass( {
+		toggleModule( name, value ) {
+			if ( 'photon' === name ) {
+				// Carousel depends on Photon. Deactivate it if Photon is deactivated.
+				if ( false === ! value ) {
+					this.props.updateOptions( { photon: false, 'tiled-gallery': false, tiled_galleries: false } );
+				} else {
+					this.props.updateOptions( { photon: true, 'tiled-gallery': true, tiled_galleries: true } );
+				}
+			} else {
+				this.props.updateOptions( { [ name ]: ! value } );
+			}
+		},
+
+		decodeEntities( text ) {
+			if ( 'undefined' === typeof DOMParser ) {
+				return text;
+			}
+
+			const document = new DOMParser().parseFromString( text, 'text/html' );
+			return document.documentElement.textContent;
+		},
+
+		render() {
+			const foundPhoton = this.props.isModuleFound( 'photon' ),
+				foundLazyImages = this.props.isModuleFound( 'lazy-images' );
+
+			if ( ! foundPhoton && ! foundLazyImages ) {
+				return null;
+			}
+
+			const photon = this.props.module( 'photon' ),
+				lazyImages = this.props.module( 'lazy-images' );
```

Comment:
We should follow Calypso's guidelines of placing each `const` definition in a new line:
```js
const photon = this.props.module( 'photon' );
const lazyImages = this.props.module( 'lazy-images' );
```